assets/app/images/replicator-icon.svg

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
First of all, for the best case, the SVG's should end up in the Patternfly font. But haven't yet gotten there.

Secondly, different users of the topology-graph component use these differently. ManageIQ wants to use PNGs.